verifier/src/tcb.rs

```diff
@@ -33,12 +33,23 @@ extern crate alloc;
 
 use alloc::string::String;
 use alloc::vec::Vec;
+use core::cmp::Ordering;
 use der::DateTime;
 use p256::ecdsa::signature::Verifier;
 use p256::ecdsa::{Signature, VerifyingKey};
 use serde::Deserialize;
 use serde_json::value::RawValue;
 
+// TODO: remove this once https://github.com/mobilecoinfoundation/sgx/pull/329
+//  and https://github.com/mobilecoinfoundation/sgx/pull/330 come in.
```

Comment:
much as this TODO says, I envision that once the type makes its way into the SGX repo then this will go away and the SGX repo types will be used.

Reply:
went away in #96